examples/express/index.js

```diff
@@ -1,6 +1,6 @@
-var app = require('express')();
+let app = require('express')();
 
-var i18n = require('i18n-future').middleware();
+let i18n = require('i18n-future').middleware();
 
 app.use(i18n);
```

Comment:
Similarly to the standalone example we can fix this by requiring `hmpo-i18n` properly, and then using the middleware correctly like so:
```
let i18n = require('hmpo-i18n');

i18n.middleware(app, {
    detect: true
});

...
```